src/sha256_armv8_neon_x4.S

```diff
@@ -277,15 +277,44 @@ TQ7		    .req q22
                     round_padding   \F, \G, \H, \A, \B, \C, \D, \E
 .endm
  
+#ifdef __APPLE__
+.global _hashtree_sha256_neon_x4
+#else
 .global hashtree_sha256_neon_x4
+#endif
+#ifdef __APPLE__
+//.type   hashtree_sha256_neon_x4,%function
```

Comment:
What's up with this?

Reply:
This is that the .type instruction doesn't exists in GNU assembler for Mac OS